### PR TITLE
Update/non refund cancellation flow CTAs

### PR DESCRIPTION
**Pull request by markryall**
This PR replaces the use of 'Remove' for products that are not eligible for a refund with 'Cancel'.

This makes the flow for cancellation of any product more consistent regardless of whether a refund is available.

# From

screen shot 2017-03-14 at 2 28 39 pm

screen shot 2017-03-14 at 2 29 39 pm

screen shot 2017-03-14 at 2 31 07 pm

# To

screen shot 2017-03-14 at 2 30 23 pm

screen shot 2017-03-14 at 2 32 41 pm

screen shot 2017-03-14 at 2 32 53 pm

### CODE REVIEW

File: client/me/purchases/remove-purchase/index.jsx [hunk 1]
```diff
@@ -362,7 +360,7 @@ const RemovePurchase = React.createClass( {
 				<p>
 					{
 						this.translate(
-							'Are you sure you want to remove %(productName)s from {{siteName/}}?',
+							'Are you sure you want to cancel %(productName)s from {{siteName/}}?',
```
Comment (a8ci18n) [inline client/me/purchases/remove-purchase/index.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/me/purchases/remove-purchase/index.jsx [hunk 2]
```diff
@@ -180,14 +179,14 @@ const RemovePurchase = React.createClass( {
...
-						this.translate( 'The domain {{domain/}} was removed from your account.', {
+						this.translate( 'The domain {{domain/}} was cancelled from your account.', {
 							components: { domain: <em>{ productName }</em> }
 						} ),
 						{ persistent: true }
 					);
 				} else {
 					notices.success(
-						this.translate( '%(productName)s was removed from {{siteName/}}.', {
+						this.translate( '%(productName)s was cancelled from {{siteName/}}.', {
```
Comment (a8ci18n) [inline client/me/purchases/remove-purchase/index.jsx]: :ok: This change is safe, we'll keep the existing 18 translations.

**Comment by alisterscott** [comment]
Tests well

**Before**

before

before step 2

**After**

after

after step 2

**Comment by gziolo** [comment]
@breezyskies @fabianapsimoes and @ranh - I wasn't involved in the v2 of Purchases implementation, so I'm not sure how it finally works. As far as I remember if purchase is non-refundable then we first cancel it which only disables auto-renewal, but keeps domain or plan active until it expires. If users want to remove cancelled purchase completely they need to go through another flow which start with `Remove purchase` navigation link. If that is still valid, then proposed changes are going to introduce even more confusion because users will have to perform two different cancel operations.

**Comment by fabianapsimoes** [comment]
@gziolo thanks for the ping!

> As far as I remember if purchase is non-refundable then we first cancel it which only disables auto-renewal, but keeps domain or plan active until it expires. 

Yep, this is correct. As I explained in p1489480634859509-upgrades-slack:

- Cancel means cancelling a subscription but leaving the product on the user's account until it expires. Basically the same as disabling auto-renewal.

- Remove is only applicable for products that have been cancelled before. It's a way to allow users to remove a product they no longer want before expiration.

I believe this change will likely increase confusion around those distinctions. Paging here @mikeshelton1503 who worked a lot with us on Purchases back then.

**Comment by markryall** [comment]
Yes - I understand it now and have managed to test each flow:

* cancellation during refund period
* cancellation after refund period with autorenew enabled
* removal when no refund is due and autorenew has been disabled

I'll close this PR now that I understand the distinction :)

Thanks for the background explanation!

**Pull request closed without merging**